docs/release-notes/currentRelease.md

```diff
@@ -1,24 +1,48 @@
 ---
-title: DevOps Docker Builds, Version 2411 (Dec 2 2024)
+title: DevOps Docker Builds, Version 2412 (Jan 6 2025)
 ---
 
-# Version 2411 Release Notes
+# Version 2412 Release Notes
 
 !!! note "Product release notes"
 For information about product changes, refer to the release notes that can be found on each
 product's [download page](https://www.pingidentity.com/en/resources/downloads.html).
 
-## DevOps Docker Builds, Version 2411 (Dec 2 2024)
+## DevOps Docker Builds, Version 2412 (Jan 6 2025)
 
 ### New Product Releases
 
-- PingFederate 12.1.3 → 12.1.4 
-- PingFederate 12.0.5 → 12.0.6
+- PingAccess 8.2.0 and EOL 8.0.x
+- PingCentral 2.2.0 and EOL 2.0.x
+- PingData products 10.2.0.0 and EOL 10.0.0.X
+    - PingDirectory ([Dockerhub](https://hub.docker.com/r/pingidentity/pingdirectory))
+    - PingDirectory Proxy ([Dockerhub](https://hub.docker.com/r/pingidentity/pingdirectoryproxy))
+    - PingDataSync ([Dockerhub](https://hub.docker.com/r/pingidentity/pingdatasync))
+    - PingAuthorize ([Dockerhub](https://hub.docker.com/r/pingidentity/pingauthorize))
+    - PingDataConsole ([Dockerhub](https://hub.docker.com/r/pingidentity/pingdataconsole))
+- PingFederate 12.2.0 and EOL 12.0.x
+
+- - PingFederate 11.3.3 → 11.3.4
```

Comment:
Duplicate lines for PA, PF ,PingCentral?

Reply:
Nice catch, forgot to remove the old